lib/i18n_alchemy/association_parser.rb

```diff
@@ -19,7 +19,7 @@ def initialize(target_class, association_name)
       #
       def parse(attributes)
         if association.macro == :has_many
-          attributes = attributes.is_a?(Hash) ? attributes.values : attributes
+          attributes = attributes.respond_to?(:to_hash) ? attributes.to_hash.values : attributes
```

Comment:
Question: won't `to_hash` raise a deprecation warning in 5.0, and an exception in 5.1, when unpermitted? I guess we might need to add some tests with AC::Parameters to be sure.